dhis-2/dhis-test-integration/src/test/java/org/hisp/dhis/tracker/export/trackedentity/TrackedEntityServiceTest.java

```diff
@@ -570,26 +573,6 @@ void shouldReturnEmptyCollectionIfGivenFilterDoesNotMatch()
     assertIsEmpty(trackedEntities);
   }
 
-  @Test
-  void shouldReturnTrackedEntitiesGivenFilterOfOnlyUidEvenIfTrackedEntityDoesNotHaveGivenAttribute()
-      throws ForbiddenException, NotFoundException, BadRequestException {
-    TrackedEntityOperationParams operationParams =
-        TrackedEntityOperationParams.builder()
-            .organisationUnits(Set.of(orgUnitA.getUid(), orgUnitB.getUid()))
-            .orgUnitMode(DESCENDANTS)
-            .trackedEntityTypeUid(trackedEntityTypeA.getUid())
-            .filters(teaA.getUid())
-            .user(user)
-            .build();
-
-    List<TrackedEntity> trackedEntities =
-        trackedEntityService.getTrackedEntities(operationParams).getTrackedEntities();
-
-    // A filter with only a UID does not cause any join on attributes, so it does not actually
-    // filter TEs from the response in contrast to /tracker/events?filterAttribute=uid
-    assertContainsOnly(List.of(trackedEntityA, trackedEntityB), trackedEntities);
-  }
-
   @Test
   void shouldReturnTrackedEntitiesEvenIfTrackedEntityDoesNotHaveGivenAttribute()
```

Comment:
These operation params do not use any attribute UID, is the test name accurate?

Reply:
I think this can be removed as well.
It was probably useful before removing `attributes` parameter